lib/src/parseable.cc

```diff
@@ -105,17 +105,38 @@ namespace hocon {
         // if it starts with "/" then remove the "/", for consistency
```

Comment:
Looks like this comment is no longer accurate.

Reply:
yeah, I'll try to correct it